### ISSUE TITLE
When the name of the tenant have not valid characteres, the error comes from express library

### ISSUE THREAD

**Issue by jcalderin** [telefonicaid/perseo-fe#25]
When a bad name of "tenant" is sent to perseo, perseo response with a 500 and the trace of the express library. It could be nice to have a parser of this errors, capture before fail, or format the failure

### The act response is:
```
HttpCode received: 500 
     Body: {"error":"error post to http://10.95.82.36:8080/perseo-core/rules (Incorrect syntax near '-' at line 1 column 24 [create context ctxt$test-34$ partition by tenant from iotEvent(tenant=\"test-34\" and service=\"/\")])","data":null}
```

**Comment by crbrox** [comment]
Fixed PR #37 
"Bad" names should not get to perseo-core

To be closed
@jcalderin @iariasleon

**Comment by iariasleon** [comment]
LGTM
